lstchain/calib/camera/pixel_threshold_estimation.py

```diff
@@ -49,30 +50,31 @@ def get_threshold_from_dl1_file(dl1_path, sigma_clean):
     picture_thresh: np.ndarray
         picture threshold calculated using interleaved pedestal events
     """
-    
+
     ped_mean_pe, ped_std_pe = get_bias_and_std(dl1_path)
 
     # If problem with interleaved pedestal std values occur, take pedestal
     # std values from calibration run.
     # Correct interleaved pedestal std array should have shape (2,2,1855)
     if ped_std_pe.shape[0] == 2:
-        interleaved_events_id = 1
+        interleaved_events_id = INTERLEAVED_CALIBRATION_ID
     else:
         interleaved_events_id = 0
```

Comment:
For consistency, you should define   ORIGINAL_CALIBRATION_ID = 0   (or whatever other name)

Reply:
Done